finalProject-2020-q3/game/player/Player.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace finalProject_2020_q3.game
+{ 
+	public class Player
+	{
+		String PlayerName { set; get; }
+		public Color PlayerColor;
+		public int PlayerIndex { set; get; }
+		public Player(string name, Color color, int index)
```

Comment:
@angelmiguelmontalvo is it necessary the index? maybe wiht the name is enough?

Reply:
Necessary to change player in next movement.